From 90699f2e5931e4bcab8b3c0c4dd037afb7619881 Mon Sep 17 00:00:00 2001
From: Pantelis Antoniou <panto@antoniou-consulting.com>
Date: Tue, 15 Jan 2013 12:11:26 +0200
Subject: [PATCH 064/177] davinci-evm: Header include move fix.

Headers have been moved. Fix by including the correct header.

Signed-off-by: Pantelis Antoniou <panto@antoniou-consulting.com>
---
 sound/soc/davinci/davinci-evm.c | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/sound/soc/davinci/davinci-evm.c b/sound/soc/davinci/davinci-evm.c
index d0fad7e..7420434 100644
--- a/sound/soc/davinci/davinci-evm.c
+++ b/sound/soc/davinci/davinci-evm.c
@@ -24,8 +24,7 @@
 #include <asm/dma.h>
 #include <asm/mach-types.h>
 
-#include <mach/asp.h>
-#include <mach/edma.h>
+#include <linux/edma.h>
 
 #include "davinci-pcm.h"
 #include "davinci-i2s.h"
-- 
1.8.1.4

